src/features/overview/components/grades/EditGradeSheet.tsx

```
@@ -0,0 +1,239 @@
+  PrimaryButton,
+  SecondaryButton,
+  Section,
+  SectionRow,
+  SelectInput,
+  Sheet,
+  Spacer,
+  TextInput,
+} from '../../../../components';
+import supabaseClient from '../../../../lib/supabase';
+
+const updateGrade = async (lsc: ILeanScopeClient, entity: Entity, updatedGrade: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)